### ISSUE TITLE
NaN breaks the integrator

### ISSUE THREAD

**Issue by kidzik** [stanfordnmbl/osim-rl#4]
When we get NaN as an action, i.e. when we set activations to NaN. We get an error of the integrator:
```
std::exception in 'bool OpenSim::Manager::integrate(SimTK::State &,double)': SimTK Exception thrown at AbstractIntegratorRep.cpp:428:
  Integrator step failed at time 0 apparently because:
SimTK Exception thrown at AbstractIntegratorRep.cpp:547:
  Error detected by Simbody method AbstractIntegrator::takeOneStep(): Unable to advance time past 0.
  (Required condition 't1 > t0' was not met.)
```
This makes sense and there should be an error but we can catch it on the python level and make the info much more meaningful.

**Comment by chrisdembia** [comment]
Simbody has "isNaN" functions, but I'm not sure if they're accessible in Python. Might need to do a contortion like creating a Vector first: `Vector(number).isNaN()` (not sure if this exists).

**Comment by kidzik** [comment]
Thanks Chris! I keep thinking about changing things on the osim-rl level, whereas I should just improve python wrappings as you mentioned.

**Comment by kidzik** [comment]
Addressed in 9846938c4bfea680b21825c3f0744736bcd5881c
